express/properties/workflow.py

```diff
@@ -1,6 +1,178 @@
+        self.obj_storage_container = object_storage_data["CONTAINER"]
+        self.obj_storage_container_region = object_storage_data["REGION"]
+        self.obj_storage_container_provider = object_storage_data["PROVIDER"]
+        self.ml_cache_dir = ".mlcache"
+        self.workflow = copy.deepcopy(self.kwargs["workflow"])
+
+    @property
+    def schema(self):
+        # Shadows the schema property in BaseProperty
+        # Not entirely sure why this is done in the old one, but the net effect is this avoids a call to
+        #   self.esse.get_property_manifest(self.name)
+        return self.esse.get_schema_by_id("workflow")
+
+    def _create_download_from_object_storage_inputs(self, filenames: List[str]) -> list:
+        """
+        Constructs the download_from_object_storage unit for use in the workflow
+
+        Args:
+            filenames: List of filenames to copy in
+            ml_cache_dir: Where the pickles are being stored
+
+        Returns:
+            A download_from_object_storage workflow unit.
+
+        """
+        inputs = []
+        if isinstance(filenames, str):
+            # Ensure we have a list of strings
+            filenames = [filenames]
+
+        for filename in filenames:
+            # If ml_cache_dir is defined (e.g. we're not just pickling to the job's root dir), make sure it's in the path
+            if self.ml_cache_dir:
+                path_names = (self.work_dir, self.ml_cache_dir, filename)
+            else:
+                path_names = (self.work_dir, filename)
+
+            inputs.append({"basename": filename,
+                           "pathname": self.ml_cache_dir,
+                           "overwrite": False,
+                           "objectData": {"CONTAINER": self.obj_storage_container,
+                                          "NAME": os.path.join(*path_names),
+                                          "PROVIDER": self.obj_storage_container_provider,
+                                          "REGION": self.obj_storage_container_region
+                                          }
+                           })
+        return inputs
+
+    def _construct_predict_subworkflows(self, train_subworkflows: list) -> list:
+        """
+        Given the set of training subworkflows, converts to the subworkflows defining the predict workflow.
+
+        Args:
+            train_subworkflows: "subworkflows" defined in the original workflow
+
+        Returns:
+            A list of subworkflows, which define the resultant predict workflow.
+
+        """
+        # Need to deepcopy to avoid changing the original subworkflow
+        predict_subworkflows = copy.deepcopy(train_subworkflows)
+
+        # Tiers need to present to make it through ESSE validations
+        tiers = {"tier1": "statistical",
+                 "tier2": "deterministic",
+                 "tier3": "machine_learning"}
+        for subworkflow in predict_subworkflows:
```

Comment:
You should not need this. The tiered classification is still WIP. We should use the version of ESSE at the commit that does not have tiers yet.

Reply:
Per conversation today (and the activity surrounding ESSE) these have been removed.